src/SwaggerProvider.DesignTime/v2/Parser/Parsers.fs

```diff
@@ -412,7 +412,20 @@ module Parsers =
             | "options" -> Some <| parseOperationObject context path Options obj
             | "head"    -> Some <| parseOperationObject context path Head obj
             | "patch"   -> Some <| parseOperationObject context path Patch obj
-            | "$ref"    -> failwith "External definition of this path item is not supported yet"
+            | "$ref"    ->
+                let fileName = obj.AsString()
+                let path =
+                    // If path is empty:
+                    // We could match something like interactive __SOURCE_DIRECTORY__
+                    // or else (System.Reflection.Assembly.GetExecutingAssembly().Location |> System.IO.Path.GetDirectoryName)
+                    obj.GetStringSafe("basePath")
+                let filePath = System.IO.Path.Combine [| path; (if fileName.Contains("#") then fileName.Split('#').[0] else fileName) |]
+
+                let schemaData =
+                    SwaggerProvider.Internal.SchemaReader.readSchemaPath "" filePath
+                    |> Async.RunSynchronously
+                
+                failwith "External definition of this path item is not supported yet"
```

Comment:
@Thorium feel questions here

1. Why to use `basePath` instead of `schemaPathRaw`? I would expect that `$ref` if relative to schema  rather than API base path
1. What is point to request schema before raising exception?

Reply:
Sorry, with this commit was an earlier commit for the other PR #152
and with that I the failwith is still the old code, the Parsers.fs changes can be ignored.